crates/bevy_transform/src/systems.rs

```diff
@@ -1,75 +1,84 @@
 use crate::components::{GlobalTransform, Transform};
-use bevy_ecs::prelude::{Changed, Entity, Query, With, Without};
+use bevy_ecs::prelude::{Changed, Entity, Or, Query, With, Without};
 use bevy_hierarchy::{Children, Parent};
 
+/// Update [`GlobalTransform`] component of entities that aren't in the hierarchy
+pub fn transform_propagate_system_flat(
```

Comment:
We don't need `_system` here.
```suggestion
pub fn sync_simple_transforms(
```